lib/linguist/heuristics.rb

```diff
@@ -380,6 +380,24 @@ def call(data)
       end
     end
 
+    disambiguate ".pod" do |data|
+      if /^\s*=\w+$/.match(data)
```

Comment:
It's possible for a Pod file to consist only of multi-word directives (e.g. =head1 NAME), which would cause this entire condition to be skipped (even after fixing the whitespace/anchoring issues). Is this supposed to disambiguate from XPM? What is that?

Reply:
@Grinnz are you referring to Pod5? So that would mean it would lack `=pod` and `=cut` designations? Is that what you mean?

If so, then I think we should add a sample which doesn't contain `=pod` or `=cut`

Reply:
The easy fix for this would be to change the `\w+` in the regex to `.+` which would match anything except newlines.

Reply:
Here's an example where the only single-word declarations are the `=back` ending the lists in the pod: https://raw.githubusercontent.com/chansen/p5-time-moment/master/lib/Time/Moment.pod

Reply:
Example with no single-word directives: https://github.com/bingos/poe-component-irc/blob/master/lib/POE/Component/IRC/Cookbook.pod (not sure if the Perl 5 license is acceptable for samples)